apps/ledger-live-desktop/tests/models/SettingsPage.ts

```diff
@@ -17,6 +24,18 @@ export class SettingsPage {
     this.experimentalTab = page.locator("data-test-id=settings-experimental-tab");
     this.experimentalDevModeToggle = page.locator("data-test-id=MANAGER_DEV_MODE-button");
     this.carouselSwitchButton = page.locator("data-test-id=settings-carousel-switch-button");
+    this.counterValueSelector = page.locator(".select__value-container").first();
```

Comment:
What happens if the order of the components changes, for example this becomes the second in the list? The test will break.

Reply:
if the order changes, it means the whole UI in this page changed, every tests related will break, not only this one

Reply:
No, the rest of the selectors are all using `data-test-id` so it will be ok (once the screenshots have updated)

Reply:
yes, for sure, if `data-test-id `works, it's the best way